packages/Avatar/src/Avatar.styles.js

```diff
@@ -4,21 +4,39 @@ import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens";
 import { getAvatarColors } from "./helpers";
 
-const smallSize = `${spacer(4)}`;
-const mediumSize = `${spacer(5)}`;
+const roundStyles = {
+  small: `
+    border-radius: 50%;
+    height: ${spacer(2.5)};
+    width: ${spacer(2.5)};
+    ${stylers.fontSize(-3)};
+  `,
+  medium: `
+    border-radius: 50%;
+    height: ${spacer(3)};
+    width: ${spacer(3)};
+    ${stylers.fontSize(-2)};
+  `,
+  large: `
+    border-radius: 50%;
+    height: ${spacer(4)};
+    width: ${spacer(4)};
+    ${stylers.fontSize(-1)};
+  `,
+};
 
-export const avatarSizeStyles = {
+const squareStyles = {
   small: `
-    border-radius: ${tokens.button.borderRadius};
-    height: ${smallSize};
-    width: ${smallSize};
+    border-radius: ${tokens.card.borderRadius};
```

Comment:
I don't know why it's saying I changed this line, cuz i didnt.  But the new one is the correct one... 🤔

Reply:
`card.borderRadius` is 6px, but here I believe we need 4px, that's why it was `tokens.button.borderRadius` before
![image](https://user-images.githubusercontent.com/38733362/101226972-15bbac00-364b-11eb-8ab0-462833ac688e.png)

Reply:
What? no, where you lookin'?

https://www.figma.com/file/DRhxHNWVzd94K8Tf0C3VTy/%F0%9F%90%A6Starling-UI-Kit?node-id=1698%3A33820

![Screen Shot 2020-12-04 at 4 13 59 PM](https://user-images.githubusercontent.com/23224777/101227209-f2453100-364b-11eb-809a-1f0476a2186e.png)

And if you inspect on Paprika storybook, it is 6px

Reply:
oh you were looking at the medium one and i'm looking at the large one

Reply:
😂 Yeah I'm looking at the small one

Reply:
We are using 6px rounded corner for 40px avatar and 4px for 32px one.